src/meta/src/hummock/manager/mod.rs

```diff
@@ -1245,11 +1245,13 @@ impl HummockManager {
         Ok(())
     }
 
+    #[named]
     pub async fn get_compact_task(
         &self,
         compaction_group_id: CompactionGroupId,
         selector: &mut Box<dyn CompactionSelector>,
     ) -> Result<Option<CompactTask>> {
+        let _timer = start_measure_real_process_timer!(self);
```

Comment:
Real process time is to measure time spent after lock is acquired but here it actually includes the lock wait time because lock is acqured in `get_compact_task_impl`

Reply:
fix it